src/libs/actions/Report.js

```diff
@@ -476,10 +482,14 @@ function fetchChatReports() {
     })
 
         // The string cast below is necessary as Get rvl='chatList' may return an int
-        .then(({chatList}) => {
+        .then((response) => {
+            if (response.jsonCode !== 200) {
+                return;
+            }
+
             // Get all the chat reports if they have any, otherwise create one with concierge
-            if (chatList.length) {
-                fetchChatReportsByIDs(String(chatList).split(','));
+            if (lodashGet(response, 'response.chatList.length')) {
```

Comment:
I think this should be `chatList.length` instead of `response.chatList.length` or else we'll only ever show concierge

Reply:
Oh yep great catch !